src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -137,6 +137,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('values', help="The task values file path relative to the source context.")
         c.argument('set_value', options_list=['--set'], help="Value in 'name[=value]' format.", action='append', validator=validate_set)
         c.argument('set_secret', help="Secret value in 'name[=value]' format.", action='append', validator=validate_set_secret)
+        c.argument('cmd', options_list=['--__cmd__'])
```

Comment:
why do you need to define two arguments for `--cmd`? Do you need to workaround some CLI limitations? If yes, can you add some comments to clarify it?

Reply:
--cmd is a default value for cmd in CLI. I am overwriting it with another value and freeing it for image use.